gpustack/worker/backends/vllm.py

```diff
@@ -214,6 +282,49 @@ def _build_command_args(self, port: int) -> List[str]:
 
         return arguments
 
+    def _build_ray_configuration(
+        self,
+        is_leader: bool,
+    ) -> (List[str], Optional[List[ContainerPort]]):
+        gcs_server_port = 6379
+        client_port = 10001
+        dashboard_port = 8265
```

Comment:
I need to further confirm that if there are any way to reduce the impact of the Ray's random port, like using container networking. This may require some additional work, so I did not delete the Ray port configuration in global config, and hard code here.